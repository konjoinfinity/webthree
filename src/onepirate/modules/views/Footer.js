import React from 'react';
import {
  MDBFooter,
  MDBContainer,
  MDBIcon,
  MDBBtn
} from 'mdb-react-ui-kit';

function Copyright() {
    return (
      <React.Fragment>
        <div style={{
                  display: "flex",
                  flexDirection: "row",
                  alignItems: "row",
                  justifyContent: "center",
                
              }}>
        <p style={{fontSize: "2vh", color: "#000"}}>{'Â©  '}</p>
        <a style={{fontSize: "2vh", color: "#000"}} href="https://reautydao.io">
          ReautyDAO
        </a>{' - '}
        <p style={{fontSize: "2vh", color: "#000"}}>{new Date().getFullYear()}</p>
        </div>
      </React.Fragment>
    );
  }

export default function Footer() {
  return (
    <MDBFooter role="footcont" className='bg-light text-center text-white'>
      <MDBContainer className='p-4 pb-0'>
        <section className='mb-4'>
          <MDBBtn
            floating
            className='m-1'
            style={{ backgroundColor: '#3b5998' }}
            href='https://reautydao.io/'
            role='button facebook'
            id='reautydao'
          >
            <MDBIcon fab icon='facebook-f' />
          </MDBBtn>
          <MDBBtn
            floating
            className='m-1'
            style={{ backgroundColor: '#55acee' }}
            href='https://twitter.com/ReautyDao'
            role='button'
          >
            <MDBIcon fab icon='twitter' />
          </MDBBtn>
          <MDBBtn
            floating
            className='m-1'
            style={{ backgroundColor: '#dd4b39' }}
            href='https://reautydao.io/'
            role='button'
          >
            <MDBIcon fab icon='google' />
          </MDBBtn>
          <MDBBtn
            floating
            className='m-1'
            style={{ backgroundColor: '#ac2bac' }}
            href='https://www.instagram.com/reautyapp/'
            role='button'
          >
            <MDBIcon fab icon='instagram' />
          </MDBBtn>
          <MDBBtn
            floating
            className='m-1'
            style={{ backgroundColor: '#0082ca' }}
            href='https://www.linkedin.com/company/blingy/'
            role='button'
          >
            <MDBIcon fab icon='linkedin-in' />
          </MDBBtn>
          <MDBBtn
            floating
            className='m-1'
            style={{ backgroundColor: '#333333' }}
            href='https://reautydao.io/'
            role='button'
          >
            <MDBIcon fab icon='discord' />
          </MDBBtn>
        </section>
             <img src={require("../views/reauty-logo.png")}
             style={{height: "10vh"}} alt="Logo"/>
      </MDBContainer>
      <Copyright />
    </MDBFooter>
  );
}