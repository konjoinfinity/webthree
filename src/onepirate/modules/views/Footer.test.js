import { render, screen } from '@testing-library/react';
import Footer from './Footer'

test('renders footer', () => {
  render(<Footer />);
  const cont = screen.getByRole("footcont");
  expect(cont).toBeInTheDocument();
});

it('footer class test', () => {
    const { container } = render(<Footer/>)
    // Assert
    expect(container).toMatchSnapshot()
    expect(container.firstChild).toHaveClass('bg-light text-center text-white')
  })

// it('should find all buttons', () => { 
//     render(<Footer/>)
//     const button = screen.getAllByRole("button")
//     expect(button).toBeInTheDocument();
// })