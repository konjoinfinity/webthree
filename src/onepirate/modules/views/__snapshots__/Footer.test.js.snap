// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer class test 1`] = `
<div>
  <footer
    class="bg-light text-center text-white"
    role="footcont"
  >
    <div
      class="container p-4 pb-0"
    >
      <section
        class="mb-4"
      >
        <a
          class="ripple ripple-surface btn btn-primary btn-floating m-1"
          href="https://reautydao.io/"
          id="reautydao"
          role="button facebook"
          style="background-color: rgb(59, 89, 152);"
        >
          <i
            class="fab fa-facebook-f"
          />
        </a>
        <a
          class="ripple ripple-surface btn btn-primary btn-floating m-1"
          href="https://twitter.com/ReautyDao"
          role="button"
          style="background-color: rgb(85, 172, 238);"
        >
          <i
            class="fab fa-twitter"
          />
        </a>
        <a
          class="ripple ripple-surface btn btn-primary btn-floating m-1"
          href="https://reautydao.io/"
          role="button"
          style="background-color: rgb(221, 75, 57);"
        >
          <i
            class="fab fa-google"
          />
        </a>
        <a
          class="ripple ripple-surface btn btn-primary btn-floating m-1"
          href="https://www.instagram.com/reautyapp/"
          role="button"
          style="background-color: rgb(172, 43, 172);"
        >
          <i
            class="fab fa-instagram"
          />
        </a>
        <a
          class="ripple ripple-surface btn btn-primary btn-floating m-1"
          href="https://www.linkedin.com/company/blingy/"
          role="button"
          style="background-color: rgb(0, 130, 202);"
        >
          <i
            class="fab fa-linkedin-in"
          />
        </a>
        <a
          class="ripple ripple-surface btn btn-primary btn-floating m-1"
          href="https://reautydao.io/"
          role="button"
          style="background-color: rgb(51, 51, 51);"
        >
          <i
            class="fab fa-discord"
          />
        </a>
      </section>
      <img
        alt="Logo"
        src="reauty-logo.png"
        style="height: 10vh;"
      />
    </div>
    <div
      style="display: flex; flex-direction: row; align-items: row; justify-content: center;"
    >
      <p
        style="font-size: 2vh; color: rgb(0, 0, 0);"
      >
        Â©  
      </p>
      <a
        href="https://reautydao.io"
        style="font-size: 2vh; color: rgb(0, 0, 0);"
      >
        ReautyDAO
      </a>
       - 
      <p
        style="font-size: 2vh; color: rgb(0, 0, 0);"
      >
        2022
      </p>
    </div>
  </footer>
</div>
`;
